ember-element-helper/package.json

```diff
@@ -44,9 +52,18 @@
     "@babel/plugin-proposal-class-properties": "7.18.6",
     "@babel/plugin-syntax-decorators": "7.18.6",
     "@babel/plugin-proposal-decorators": "7.18.6",
-    "@embroider/addon-dev": "1.8.3",
+    "@babel/preset-typescript": "7.22.5",
+    "@babel/runtime": "^7.22.6",
+    "@embroider/addon-dev": "3.2.0",
```

Comment:
can we up to addon-dev @v4? :sweat_smile: was just released this morning :partying_face: